src/ActionsImporter/Constants.cs

```diff
@@ -26,6 +26,7 @@ public static class Constants
         new Variable("TRAVIS_CI_INSTANCE_URL", Provider.TravisCI, "Base url of the Travis CI instance", "https://travis-ci.com"),
         new Variable("TRAVIS_CI_ORGANIZATION", Provider.TravisCI, "Travis CI organization name"),
         new Variable("BITBUCKET_ACCESS_TOKEN", Provider.Bitbucket, "Personal access token for Bitbucket"),
+        new Variable("CONTAINER_REGISTRY", Provider.Docker, "Container registry to use for the actions-importer image", "ghcr.io"),
```

Comment:
I was thinking that it'd be nice to have a way to discover/configure this environment variable. So I added Docker as a provider which results in the configure command having a "Docker" option. And then when you select it you see this:

![Screenshot 2023-10-03 at 11 22 08](https://github.com/github/gh-actions-importer/assets/13201458/a26ebb97-416c-4e47-96c1-b9c1ff9221bb)

What do folks think of that? It's a little confusing, but I don't see another super easy way to reveal this field. I can also add something to the README, but looking for feedback on this or for entirely different ideas!

Reply:
I'd prefer to have this be a small call out in the readme instead of being configurable through this prompt. We want to steer customers _away_ from doing this if possible, since it makes it harder to support and provide updates to users 💭

Reply:
Good point! Will remove the Docker configure items and add a blurb to the README.